vrs/marten-construction-script.py

```diff
@@ -1,168 +1,180 @@
-#!/usr/bin/env python
-# coding: utf-8
-
-# Daniel Marten 01-04-2022
-# Constructing a ~100k Variant Test Database for GA4GH-VRS Annotation Script
-# NOTE: adapted from a Jupyter Notebook, so excuse any formatting quirks! 
-# NOTE: please make sure to change the paths for checkpointing!!!
-# Edits on: 01-06-2022
+# Daniel Marten 01-20-2023
+# Construct a ~100k Variant Test VCF for GA4GH-VRS Project
+# Reach out to me at: marten@broadinstitute.org
```

Comment:
```suggestion
"""
Script that constructs a ~100k Variant Test VCF for GA4GH-VRS Project

Contact: Daniel Marten (marten@broadinstitute.org)
"""
```
in general, our team uses this docstring format to add descriptions at the top of scripts. also, as a minor note here, GitHub has tracked the history of this script (so it should be fairly clear you wrote this), though adding a contact email is a nice touch